src/main/java/org/opensearch/security/auditlog/impl/AuditMessage.java

```diff
@@ -417,7 +417,7 @@ void addRestRequestInfo(final SecurityRequest request, final AuditConfig.Filter
                     } else {
                         auditInfo.put(REQUEST_BODY, requestBody);
                     }
-                } catch (IOException e) {
+                } catch (Exception e) {
                     auditInfo.put(REQUEST_BODY, "ERROR: Unable to generate request body");
```

Comment:
should we log the actual exception for troubleshooting/debugging?

Reply:
Added. Will raise new revision